packages/synapse-react-client/src/components/Authentication/OneTimePasswordForm.test.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react'
+import { SynapseContextType } from '../../utils'
+import { render, screen } from '@testing-library/react'
+import { createWrapper } from '../../testutils/TestingLibraryUtils'
+import userEvent from '@testing-library/user-event'
+import OneTimePasswordForm, {
+  BEGIN_RESET_2FA_BUTTON_TEXT,
+  OneTimePasswordFormProps,
+  SEND_RESET_2FA_EMAIL_BUTTON_TEXT,
+  SHOW_RECOVERY_CODE_FORM_BUTTON_TEXT,
+  SHOW_TOTP_FORM_BUTTON_TEXT,
+} from './OneTimePasswordForm'
+import { MOCK_CONTEXT_VALUE } from '../../mocks/MockSynapseContext'
+
+const defaultProps = {
+  step: 'VERIFICATION_CODE',
+  hideReset2FA: false,
+  loginIsPending: false,
+  onClickPromptReset2FA: jest.fn(),
+  onClickReset2FA: jest.fn(),
+  onClickUseBackupCode: jest.fn(),
+  onClickUseTOTP: jest.fn(),
+  onSubmit: jest.fn(),
+  twoFactorAuthResetIsPending: false,
+  twoFactorAuthResetIsSuccess: false,
+} satisfies OneTimePasswordFormProps
+
+function renderComponent(
+  props: OneTimePasswordFormProps,
+  wrapperPropOverrides?: Partial<SynapseContextType>,
+) {
+  return render(<OneTimePasswordForm {...props} />, {
+    wrapper: createWrapper({ ...MOCK_CONTEXT_VALUE, ...wrapperPropOverrides }),
+  })
+}
+
+function setUp(
+  props: OneTimePasswordFormProps = defaultProps,
+  wrapperPropOverrides?: Partial<SynapseContextType>,
+) {
+  const user = userEvent.setup()
+  const component = renderComponent(props, wrapperPropOverrides)
+  return { component, user }
+}
+describe('OneTimePasswordForm', () => {
```

Comment:
Testing backfill + new scenarios for this OneTimePasswordForm component. These tests just ensure that certain props map to the correct UI state, and certain actions invoke the correct callbacks.